### DISCUSSION TITLE
Thank you <3

### DISCUSSION THREAD

**Discussion by Dovahkun** [Monika-After-Story/MonikaModDev#8827] [appreciation]
Just wanted to thank everyone so much for making this mod as amazing as it is. It's come so far and I have the upmost respect for those spending hours upon hours (days upon days. months upon months ofc) making it the best it can be. From the suggestions leading up to the dialogue, all that the coding, and ahhhhhhh the art <3 (Amazing outfit for today btw! I've been praying for the casual look and love that you waited for the _perfect_ day) You all are so. so. **_SO_** wonderful and I love you guys ^_^ Have an awesome Valentine's day all!

**Comment by ghost** [comment]
Seconded, much love everyone. This mod is the best!
